Makefile

```diff
@@ -9,7 +9,7 @@ help:
 %:
 	make -f common/Makefile $*
 
-install: deploy ## installs the pattern, inits the vault and loads the secrets
+install: operator-deploy ## installs the pattern, inits the vault and loads the secrets
```

Comment:
Can we split this change out in a separate PR? It's really unrelated to this work